sql/rpl_gtid_state.cc

```diff
@@ -878,23 +878,17 @@ void Gtid_state::update_gtids_impl_own_gtid(THD *thd, bool is_commit) {
     In Group Replication the GTID may additionally be owned by another
     thread, and we won't remove that ownership (it will be rolled back later)
   */
-<<<<<<< HEAD
 #ifdef WSREP
```

Comment:
Shouldn't this be `#ifdef WITH_WSREP`?

Reply:
Maybe. But its there for a long time, and I didn't want to change it as part of a merge pr.